profile.php

```diff
@@ -102,6 +109,9 @@
 $deleteallbutton = new \single_button($deleteallurl, get_string('deleteprofiles_script', 'tool_excimer'));
 $deleteallbutton->add_confirm_action(get_string('deleteprofiles_script_warning', 'tool_excimer'));
 
+$lockprofileurl = new \moodle_url('/admin/tool/excimer/lock_profile.php', ['profileid' => $profileid]);
+$lockprofilebutton = new \single_button($lockprofileurl, 'Lock/Unlock', 'GET');
```

Comment:
1) should be lang string
2) Can you make this say 'Lock' when it is not locked and Unlock with it is locked